parm/config/gfs/config.stage_ic

```diff
@@ -9,10 +9,19 @@ source "${EXPDIR}/config.resources" stage_ic
 
 case "${CASE}" in
   "C48" | "C96" | "C192")
-    export CPL_ATMIC="workflow_${CASE}_refactored"
-    export CPL_ICEIC="workflow_${CASE}_refactored"
-    export CPL_OCNIC="workflow_${CASE}_refactored"
-    export CPL_WAVIC="workflow_${CASE}_refactored"
+
+    if [ -z ${PW_CSP+x} ]; then
+      export CPL_ATMIC="workflow_${CASE}_refactored"
+      export CPL_ICEIC="workflow_${CASE}_refactored"
+      export CPL_OCNIC="workflow_${CASE}_refactored"
+      export CPL_WAVIC="workflow_${CASE}_refactored"
+    else
+      echo "PW_CSP: $PW_CSP"
+      export CPL_ATMIC="${CASE}"
+      export CPL_ICEIC="${CASE}"
+      export CPL_OCNIC="${CASE}"
+      export CPL_WAVIC="${CASE}"
+    fi
```

Comment:
Why is the canned ICs stored in a different directory naming structure on the cloud? This seems unnecessary